<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectTask</class>
 <widget class="QDialog" name="SelectTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Task</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>865</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>951</width>
      <height>621</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tryStiff">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Start Trial!</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Select the task to start the test</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="tryLetter">
     <property name="geometry">
      <rect>
       <x>625</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Start Trial!</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>220</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stiffness Discrimination</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Bitte w√§hlen Sie die Aufgabe zum Starten des Tests</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="StartLetterButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>530</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;START&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>180</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Letter Writing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="StartStiffnessButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>530</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>521</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>tryStiff</zorder>
    <zorder>frame_5</zorder>
    <zorder>label_4</zorder>
    <zorder>tryLetter</zorder>
    <zorder>frame_2</zorder>
    <zorder>label</zorder>
    <zorder>StartLetterButton</zorder>
    <zorder>StartStiffnessButton</zorder>
    <zorder>frame_4</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
